responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html

```diff
@@ -297,7 +297,7 @@
             </div>
           </div>
 
-          <div class="row">
+          <div class="row" *ngIf="canCloneSupport()">
```

Comment:
did you mean canEditSupport?

Reply:
I've created 2 function - canEdit and canClone support as in future they could have different conditions. 
CanEdit is in this code as well, but checks only Edit button. And canClone - is a condition for  3 'Clone & Extend' buttons

Reply:
canCloneOrExtend? Probably better separate the functions and still have the same logic in them for a future maintainer

Reply:
created 2 functions, and used both.